index.html

```diff
@@ -0,0 +1,150 @@
+          <li class="nav-item"><a href="./about.html">About </a></li>
+        </ul>
+      </nav>
+      <div class="hero-container">
+        <div class="hero">
+          <p class="hero-text">"Hello! Sharing World"</p>
+          <h1 class="hero-leading-text">
+            Pan African Conference on Artificial Intelligence 2022
+          </h1>
+          <h5 class="hero-description">
+            A key topic in digitization and industry 4.0. in line with global
+            trends, the Middle East and Africa expect a 400% turnover increased
+            by 2025. AI is a UNESCO development priority in 21 of 32 African
+            countries surveyed.
+          </h5>
+          <h3 class="hero-date">2022.10.4(THU) ~ 5(FRI)</h3>
+          <span>@Ethiopia Science Museum, Addis Ababa Ethiopia</span>
+        </div>
+      </div>
+    </header>
+    <main>
+      <section class="main-program">
+        <h2 class="main-program-title heading-text title">Main Program</h2>
+        <ul class="programs">
+          <li class="program">
+            <a href="#"><i class="fa-brands fa-airbnb fa-2xl"></i></a>
+            <h2 class="program-type">Lecture</h2>
+            <p class="program-description">
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Ullam,
+              ducimus?
+            </p>
+          </li>
+          <li class="program">
+            <a href="#"><i class="fa-brands fa-shopify fa-2xl"></i></a>
+            <h2 class="program-type">CC Exhibition</h2>
+            <p class="program-description">
+              Lorem ipsum dolor sit amet consectetur adipisicing elit.
+              Voluptate, modi.
+            </p>
+          </li>
+          <li class="program">
+            <a href="#"><i class="fa-brands fa-snapchat fa-2xl"></i></a>
+            <h2 class="program-type">Forum</h2>
+            <p class="program-description">
+              Lorem ipsum dolor, sit amet consectetur adipisicing elit. Tempora,
+              accusantium.
+            </p>
+          </li>
+          <li class="program">
+            <a href="#"><i class="fa-sharp fa-solid fa-shop fa-2xl"></i></a>
+            <h2 class="program-type">Workshop</h2>
+            <p class="program-description">
+              Lorem ipsum dolor, sit amet consectetur adipisicing elit. Tempora,
+              accusantium.
+            </p>
+          </li>
+          <li class="program">
+            <a href="#"><i class="fa-brands fa-meta fa-2xl"></i></a>
+            <h2 class="program-type">CC Ignite</h2>
+            <p class="program-description">
+              Lorem ipsum dolor, sit amet consectetur adipisicing elit. Tempora,
+              accusantium.
+            </p>
+          </li>
+        </ul>
+        <button class="join-btn btn btn-secondary">Join PanAfrica 2022</button>
+        <a href="#" class="see-whole-program-btn">See The Whole Program</a>
+      </section>
+      <section class="feature-speaker">
```

Comment:
- **Desktop: Home**: It contains a bottom section with a column that includes **six individual articles, each with an image,**
- Kindly remove some and make it six (6):

![26](https://user-images.githubusercontent.com/63972670/203529398-5e40ee56-c180-48d7-a07a-86bbab8eb1ef.PNG)